examples/node-agent-11labs/index.js

```
@@ -0,0 +1,261 @@
+const { writeFile, appendFile } = require("fs/promises");
+const { createClient, AgentEvents } = require("../../dist/main/index");
+const fetch = require("cross-fetch");
+const { join } = require("path");
+
+const deepgram = createClient(process.env.DEEPGRAM_API_KEY);
```

ESLint

[error] 'process' is not defined.

(no-undef)